Help/MAC_Hints.rst

```diff
@@ -1,25 +1,36 @@
 Coverage Installation Hints for macOS Users:
 ============================================
 
+venv
+----
+
+Here we will be using venv, which is part of python's standard
+libary since python 3.3, to create a virtualenv for development.
+
 1. Make sure you have installed Xcode and Homebrew.
----------------------------------------------------
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 2. Install Python3.
--------------------
+~~~~~~~~~~~~~~~~~~~
+
+For coala you will need to use python3, so you may
```

Comment:
in texts please use Python 3 consistently, uppercase P, space before the 3. Appears multiple times wrongly across your commit